docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,27 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting and Getting the Server Icon URI
+
+To retrieve the server icon URI from the client, the server must set the URI in advance.
+If you try to read the server icon URI before it is set, the result is a null value.
+To retrieve the server icon URI from the client, follow these steps:
+
+1. Set the server icon on the server side:
+
+    ```
+    var server = tizen.mediacontroller.createServer();
+    server.updateIconURI("icon.png");
+    ```
+
+2. Get the server icon URI on the client side:
+
+    ```
+    var client = tizen.mediacontroller.getClient();
+    var server_info = client.getLatestServerInfo();
+    console.log(server_info.iconURI);
+    ```
+
 ## Receiving Notifications from the Server
```

Comment:
Global comment: To maintain consistency, please avoid articles from heading.
Receiving Notifications from Server

Please fix the same in line number: 62, 282, and 308.